src/Microsoft.Health.Fhir.Shared.Web/appsettings.json

```diff
@@ -182,7 +182,9 @@
         "ParallelQueryOptions": {
             "MaxQueryConcurrency": 500,
             "EnableConcurrencyIfQueryExceedsTimeLimit": true
-        }
+        },
+        "LogSDKDiagnostics": false,
```

Comment:
We've mostly camelcased acronyms in the codebase too - `LogSdkDiagnostics`

Reply:
Changed